examples/mixture/plot_dirichlet_process_mixture.py

```diff
@@ -0,0 +1,115 @@
+"""
+=============================================
+Dirichlet Process Mixture Beta Prior Analysis
+=============================================
+
+Plot the resulting ellipsoids of a mixture of three Gaussians with
+a Dirichlet Process Gaussian Mixture for three different values of the
+beta concentration prior.
+
+For all models, the Dirichlet Process Gaussian Mixture adapts its number of
+mixture automatically. The parameter `beta_concentration_prior` has a
+direct link with the resulting number of components. Specifying a high value of
+`beta_concentration_prior` leads more often to uniformly-sized mixture
+components, while specifying small values will lead to some mixture
+components getting almost all the points while most mixture components will be
+centered on just a few of the remaining points.
+"""
+# Author: Thierry Guillemot <thierry.guillemot.work@gmail.com>
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+from sklearn.mixture import DirichletGaussianMixture
+
+print(__doc__)
+
+
+def plot_ellipses(ax, weights, means, covars):
+    for n in range(means.shape[0]):
+        v, w = np.linalg.eigh(covars[n][:2, :2])
```

Comment:
Isn't the notation `w` used to denote the eigen values and `v` for vectors and hence should it be `w, v`?